apps/onyx/lib/core/widgets/double_scrollable_widget.dart

```diff
@@ -0,0 +1,151 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+/// don't forget to add NeverScrollableScrollPhysics() to all the scrollable widgets
```

Comment:
Did you forget or is it good ?

Reply:
It's is a note for using this widget. So it is good.